packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [51.0.2]
+
+### Uncategorized
```

Comment:
Don't think this should say "Uncategorized"

Reply:
good callout , it should say fixed